backend/accounts/models.py

```diff
@@ -133,3 +133,53 @@ class PhoneNumber(models.Model):
 
     def __str__(self):
         return f"{self.user} - {self.value}"
+
+
+class PrivacyResource(models.Model):
+    """
+    Represents a resource utilized by Penn Labs that users reserve
+    the right to withhold.
+    """
+
+    name = models.CharField(max_length=255)
+
+
+@receiver(post_save, sender=PrivacyResource)
+def add_privacy_resource(sender, instance, created, **kwargs):
+    """
+    This post_save hook triggers whenever a new privacy resource is added.
+    Each User will receive a new privacy setting with this resource, enabled
+    to true.
+    """
+    users = User.objects.all()
+    resources = [None] * users.count()
+    for i, user in enumerate(users):
+        resources[i] = PrivacySetting(user=user, resource=instance, enabled=True)
```

Comment:
Would this also work? 

```py
resources = [PrivacySetting(user=user, resource=instance, enabled=True) for user in users]
```

Reply:
Oh wow I just looked it up and yeah I thought pre-allocating the size of the list would be more efficient, but turns out list comprehension does in under the hood as well. Thanks!